providers/class.two-factor-totp.php

```diff
@@ -80,12 +80,13 @@ public function user_two_factor_options( $user ) {
 		<?php if ( empty( $key ) ) :
 			$key = $this->generate_key();
 			$site_name = get_bloginfo( 'name', 'display' );
+			$totp_title = apply_filters( 'two_factor_totp_title', $site_name . ':' . $user->user_login );
```

Comment:
Could we pass `$user` as the second parameter to the `two_factor_totp_title` filter? That would simplify user related lookups during the filter.

Reply:
Yeah, I was thinking about that too. Just so I'm clear are you thinking something like:

`apply_filters( 'two_factor_totp_title', $site_name . ':' . $user->user_login¸ $user );`

Reply:
Yes, exactly.